chain-vote/src/unit_vector.rs

```diff
@@ -28,13 +28,17 @@ impl UnitVector {
     }
 
     pub fn iter(&self) -> UnitVectorIter {
-        UnitVectorIter(0, self.clone())
+        UnitVectorIter(0, *self)
     }
 
     pub fn len(&self) -> usize {
         self.size
     }
 
+    pub fn is_empty(&self) -> bool {
```

Comment:
This is why I'm pathologically against "satisfying the linter" as .. `is_empty` cannot ever be useful in the case of unit_vector, as the size will always be `> 0` as enforced in `new()`.

Reply:
Makes sense though. I'll just disable the warning and add this explanation.